package/network/utils/umbim/files/lib/netifd/proto/mbim.sh

```diff
@@ -294,6 +295,19 @@ _proto_mbim_setup() {
 		fi
 	}
 
+	[ -z "$mtu" ] && {
+		local ipv4mtu=$(_proto_mbim_get_field ipv4mtu "$mbimconfig")
+		ipv4mtu="${ipv4mtu:-0}"
+		local ipv6mtu=$(_proto_mbim_get_field ipv6mtu "$mbimconfig")
+		ipv6mtu="${ipv6mtu:-0}"
+
+		mtu=$((ipv6mtu > ipv4mtu ? ipv6mtu : ipv4mtu))
+	}
+	[ -n "$mtu" -a "$mtu" != 0 ] && {
+		echo Setting MTU of $ifname to $mtu
+		/sbin/ip link set dev $ifname mtu $mtu
```

Comment:
Why do you set the mtu manually and do not create a json message?

netifd supports a mtu and a mtu6.

Reply:
I based that off of implementation for uqmi. I'm trying to figure out where to add the `mtu` parameter to JSON, as for other examples I've seen it only in `tunnel` section, or so it seems - in conjunction with call to `proto_add_tunnel`.

Reply:
Okay - I figured it out - this attribute is normally paired with `config device`, so I would have to update this with separate message, so netifd could pick it up.

Reply:
ok this is fine with me.